config/gpu_configs/gpu_hs_rhoe_equilmoist_nz63_0M_55km_rs35km.yml

```diff
@@ -1,12 +1,12 @@
 dt_save_to_disk: "10days"
-dt: "150secs"
-t_end: "300days"
-h_elem: 16
+dt: "100secs"
+t_end: "1days"
```

Comment:
I suggest we go at least 10 days, so that the callbacks are exercised.

Reply:
what callbacks need to be exercised?

Reply:
The default are probably fine, which will probably include a bunch of model-specific pieces (e.g., diagnostics for moisture etc.).

Reply:
can we just reduce the `dt_save_to_disk`?

Reply:
okay, but that doesn't need to be in the perf target. We should make sure it is in the CI though.

Reply:
> can we just reduce the dt_save_to_disk?

Yeah, that works too